test/e2e/app-dir/next-form/default/next-form.test.ts

```diff
@@ -0,0 +1,190 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('app dir - form', () => {
+  const { next, skipped, isNextDev } = nextTestSetup({
+    files: __dirname,
+    skipDeployment: true,
```

Comment:
wondering if we need to skip deployment here?

Reply:
no reason, leftover from old version. removed